site/templates/default.html

```diff
@@ -12,6 +12,38 @@
       <link rel="stylesheet" href="/css/hl.css">
       <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/glider-js@1/glider.min.css">
       <script src="https://cdn.jsdelivr.net/npm/glider-js@1/glider.min.js"></script>
+      <script>
+         (function () {
+            try {
+              var preferredUwu;
+              try {
+                preferredUwu = localStorage.getItem('uwu');
+              } catch (err) { }
```

Comment:
`localStorage.getItem` doesn't throw exception, if memory serves well. Why the try-catch block?

Reply:
Indeed, it just returns null.

Reply:
Not that it matters but I believe it is possible to disable localStorage under certain privacy settings, then it will throw something like `Uncaught DOMException: Failed to read the 'localStorage' property from 'Window': Access is denied for this document.` in Chrome. You can try it yourself in Chrome by changing the settings at Settings > Privacy and security > Site settings > Additional content settings > On-device site data > Don't allow sites to save data on your device

Reply:
Indeed, but that should be caught by the handler around the whole function, right?

Reply:
> Indeed, but that should be caught by the handler around the whole function, right?

The idea is that we still want to set the class name regardless whether we have successfully read the localStorage value. i.e. if `?qwq=true` is set in the url, then the logo is guaranteed to be switched, no matter if the localStorage is working or not.